fuzzers/074-dump_all/create_node_tree.py

```diff
@@ -0,0 +1,261 @@
+import argparse
+import datetime
+import progressbar
+import json
+import os.path
+import reduce_tile_types
+import pickle
+import collections
+
+def build_node_index(fname):
+  node_index = {}
+  with open(fname, 'rb') as f:
+    f.seek(0, 2)
+    bytes = f.tell()
+    f.seek(0, 0)
+    with progressbar.ProgressBar(max_value=bytes) as bar:
```

Comment:
haven't seen this library before, will have to check it out. Fuzzers could use this instead of my diy solution